sql-files/upgrades/upgrade_20230915.sql.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS `macro_user_report` (
+  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+  `reporter_aid` INT UNSIGNED NOT NULL,
+  `reported_aid` INT UNSIGNED NOT NULL,
+  `report_type` INT UNSIGNED NOT NULL,
+  `report_msg` VARCHAR(256) NOT NULL DEFAULT '',
```

Comment:
Why define it as 256 here, when it is actually 101?

Reply:
I don't know either. The original file came like this.